### ISSUE TITLE
ammo packs reload infinitely on RC builds

### ISSUE THREAD

**Issue by MoerasGrizzly** [scp-fs2open/fs2open.github.com#5080]
@BMagnu Bug has not fully been fixed, a new issue has arisen instead: When firing your primary weapons in Her Finest Hour (for instance), all ammo is instantly replenished as if an ammo pack were used. However, the ammo pack is not selected by default at the start of the mission, and the ammo pack itself is not used up.

Weirdly enough, cycling through your primary weapon selections even once prevents the issue from occuring. But as long as you don't touch the period key at all, you get infinite ammo on both your sidhe and grimmlers.

I tested this with 22.2 stable and it does not occur there.

_Originally posted by @MoerasGrizzly in https://github.com/scp-fs2open/fs2open.github.com/issues/4952#issuecomment-1352347291_

**Comment by MjnMixael** [comment]
I tested this out on the Jan 4 nightly and BP Complete 3.1.6 following these steps to reproduce and could not encounter the issue as described.

1: Load up BP, switch to the R2 campaign and load Her Finest Hour from the Tech Room
2: Go into mission with the default loadout
3: Without changing weapons, begin firing primaries and secondaries (This is where I expected to encounter the bug of Ammo Pack running)
4: Ammo pack did not run for my tests, so I switched to Ammo Pack and fired the weapon. My ammo was restored and Ammo Pack was depleted as expected.
